src/form/repeatable.tsx

```diff
@@ -0,0 +1,145 @@
+import * as React from 'react';
+
+export interface RepeatableStatelessProps {
+    titleRepeat: string;
+    repeatedItems: Array<any>;  // child form elements
```

Comment:
We probably could make this of type `Array<{ [idx: string]: any }>`, and then not require the dataKeys props (deriving it ourselves inside the component)